docs/content/docs/test/Toast.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Testing toasts
+type: docs
+---
+
+# Testing toasts
+
+[android.widget.Toast](https://developer.android.com/reference/android/widget/Toast)
+
+{{< hint warning>}}
+Testing toasts reliably is a hard task because some other toast can show up and yours will be queued. 
+We hack app with proxy object to mitigate this.
+{{< /hint >}}
+
+{{< hint warning>}}
+It is also recommended to use [snackbar](https://developer.android.com/reference/com/google/android/material/snackbar/Snackbar), 
+or your custom way to display brief unintrusive messages instead. So you should consider not testing it at all.
+{{< /hint >}}
+
+## Proxy toast
+
+Use provided extension methods to be able to spy on `showToast` functions.
+
+```kotlin
+import com.avito.android.util.showToast
+
+showToast("I'am a toast!")
+```
+
+```kotlin
+implementation("com.avito.android:proxy-toast:$version")
```

Comment:
Do we need describe all deps?

Reply:
what deps? it's enough for this library to run

Reply:
I understand nothing.
You created too modules `proxy-toast` and `toast-rule`. You added them to test-app dependencies.
Is `proxy-toast` enough for what without toast-rule?
You described usages of ToastRule but didn't add description how to add it dependency. Comparing with `proxy-toast` why didn't you add dependency description?

Reply:
yep, i missed that, thanks